src/protocols/output_management.rs

```diff
@@ -114,15 +114,41 @@ impl OutputManagementManagerState {
                 let modes_changed = old.modes != conf.modes;
```

Comment:
Need to update comment above this line saying that custom modes add/remove a mode on the TTY